cmd/entrypoint/namespaces_test.go

```diff
@@ -15,6 +15,18 @@ import (
 // sandboxed environment, the test could pass even if the dropNetworking
 // function did nothing.
 func TestDropNetworking(t *testing.T) {
+	// First make sure we can run the dropNetworking command.
+	// Some older kernels require special configurations to run this.
+	// I haven't been able to come up with an exhaustive list of what is needed,
+	// but it includes things like CAP_SYS_ADMIN, kernel.unprivileged_userns_clone=1
+	// and maybe others.
+	// For the sake of this test just check it first.
+	testCmd := exec.Command("true")
+	dropNetworking(testCmd)
```

Comment:
If there is an issue with dropNetworking() would this always be Skip then  and make the test not working ? Maybe filtering on the type of error ?

Reply:
I think we could even filter on the capabilities available and/or on the user uid that executes the test. As is, it's a bit weird because if the function fails for another reason than not having enough capabilities to remove capabilities (oh the irony 😛 ), it's gonna skip instead of fail.